integration-tests/tests/network/routing.rs

```diff
@@ -103,6 +103,8 @@ fn ping_jump() {
     runner.push(Action::AddEdge(0, 1));
     runner.push(Action::AddEdge(1, 2));
     runner.push(Action::CheckRoutingTable(0, vec![(1, vec![1]), (2, vec![1])]));
+    runner.push(Action::CheckRoutingTable(1, vec![(0, vec![0]), (2, vec![2])]));
```

Comment:
comment about line 100:

can you add the comment to the test, describing what it is testing

Reply:
I added comments.